src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -170,10 +164,8 @@ private String extractRequestBody(StaplerRequest req) {
         return body;
     }
 
-    
-    private Set<GhprbWebHook> getWebHooks() {
-        final Set<GhprbWebHook> webHooks = new HashSet<GhprbWebHook>();
-
+    private Set<GhprbTrigger> getTriggers(String repoName, String body, String signature) {
+        final Set<GhprbTrigger> triggers = new HashSet<GhprbTrigger>(5);
```

Comment:
I think I would initially set this to null and then new it only if you get a matching trigger.  The vast majority of comments that appear on GH aren't trigger phrases, so we'd end up with a lot of unneeded allocations.